### PR TITLE
Feature/location

### PR DESCRIPTION
**Pull request by ChristianJoe**

### CODE REVIEW

File: handlers/location.js [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    let messageText = ctx.dialogflowResponse;
+    const zipCode = location['zip-code'].stringValue;
+    let city = location.city.stringValue;
+    console.log(`zipCode: ${zipCode}`);
+    console.log(`city: ${zipCode}`);
+    if (byZipCodes[zipCode]) {
+        city = byZipCodes[zipCode].city;
+    }
+    if (byCities[city]) {
+        return handleCity(ctx, byCities[city]);
```
Comment (jh0ker) [inline handlers/location.js]: you are returning here before the tracking could happen. on purpose?

File: handlers/location.js [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+            campaignId: 'covid',
+        });
+    const ddjLink = `\n🔗 <a href="${escapeHTML(ddjUrl)}">${
+        escapeHTML(`Aktuelle Zahlen zur Corona-Krise in NRW`)
+    }</a>`;
+
+    const messageText = `Hier die aktuellen Zahlen für ${
+        cityFull.city
+    }${
+        cityFull.keyCity.slice(-1) === '0' ? '' : ' im Landkreis ' + cityFull.district
```
Comment (jh0ker) [inline handlers/location.js]: gleiche formulierungs-frage wie im fb-bot

File: handlers/location.js (hunk 1 above)
Comment (ChristianJoe) [reply handlers/location.js]: copy-paste error - fixed

File: handlers/location.js (hunk 2 above)
Comment (ChristianJoe) [reply handlers/location.js]: fixed

**Pull request merged**